components/courses/app/services/courses/test_task/update.rb

```diff
@@ -6,8 +6,8 @@ class Update < ApplicationService
       def initialize(test_task, mentor, status: Courses::TestTask::ON_REVIEW, notes: nil)
```

Comment:
`def initialize(test_task, mentor, params = {})`

and just handle 1 argument: params